lib/beacon/component_categories.ex

```diff
@@ -0,0 +1,49 @@
+defmodule Beacon.ComponentCategories do
+  @moduledoc """
+  The ComponentCategories context.
+  """
+
+  import Ecto.Query, warn: false
+  alias Beacon.Content.ComponentCategory
+
+  alias Beacon.Repo
+
+  @doc """
+  Returns the list of component categories.
+
+  ## Examples
+
+      iex> list_component_categories()
+      [%ComponentCategories{}, ...]
+
+  """
+  def list_component_categories do
+    ComponentCategory |> Repo.all()
```

Comment:
```suggestion
    Repo.all(ComponentCategory)
```
No need to use the pipe operator if there is only one